packages/interaction/src/IHitArea.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Interface for classes that represent a hit area.
+ *
+ * @interface IHitArea
+ * @memberof PIXI
+ */
+
+/**
+ * Checks whether the x and y coordinates given are contained within this Rectangle
```

Comment:
Within this shape, rather than rectangle

Reply:
Thank you for the review. As pointed out, the `rectangle` is not appropriate. Because this is a `IHitArea` interface, I think the word `area` fits well. Do you have any thoughts?

Reply:
yeps, area sounds good

Reply:
I updated the JSDoc comment as per the review.